tests/RuleSetTest.php

```diff
@@ -228,6 +228,39 @@ public function providerSetDefinitionNames()
         }, $setDefinitionNames);
     }
 
+    /**
+     * @param array $set
+     * @param bool  $safe
+     *
+     * @dataProvider provideSafeSets
+     */
+    public function testRiskyRulesInSet(array $set, $safe)
+    {
+        $this->assertSetSafety($set, $safe);
```

Comment:
now this test method is just a wrapper for private method. please remove that private one